Examples/Gdip.Tutorial.11-Using.BRAs.to.perform.animations.ahk

```diff
@@ -97,15 +97,16 @@ If (pBitmap < 1)
 
 ; Calculate the dimensions of the gui. I advise you always work on fractions of the screen dimensions to ensure it will look similar on other screens
 ; If you feel like it, you could go further and change the dimensions based on 16:9 or 4:3
-WinWidth := WAWidth//2.5
+WinWidth := Floor(WAWidth/2.5)
 WinHeight := Round(WinWidth//4)
 
 ; Get the width and height of a single bitmap. This bitmap is the fish banner
 Width := Gdip_GetImageWidth(pBitmap), Height := Gdip_GetImageHeight(pBitmap)
 
 ; Calculate the dimensions the fish banner will be resized to, to fit nicely on the gui
 NewWidth := 0.9*WinWidth
-NewHeight := Round((NewWidth/Width)*Height)
+NewHeight := (NewWidth/Width)*Height
+NewMargin := (WinWidth-NewWidth)/2
```

Comment:
all the functions here seem fine with floats, so I assume no need to round (already NewWidth was not an integer)